save-backend/src/main/kotlin/com/saveourtool/save/backend/service/ProjectService.kt

```diff
@@ -118,9 +120,13 @@ class ProjectService(
     /**
      * @param name
      * @param organizationName
+     * @param status
      * @return project
      */
-    fun findByNameAndOrganizationName(name: String, organizationName: String) = projectRepository.findByNameAndOrganizationName(name, organizationName)
+    fun findByNameAndOrganizationName(name: String, organizationName: String, status: ProjectStatus = ProjectStatus.CREATED) =
+            projectRepository.findByNameAndOrganizationName(name, organizationName)?.takeIf {
+                it.organization.status == OrganizationStatus.CREATED && it.status == status
```

Comment:
so it means that DELETED project can be found only in non-deleted organization?
Didn't get this checking of organization status